be/src/olap/field.h

```diff
@@ -55,9 +55,11 @@ class Field {
     inline size_t field_size() const { return size() + 1; }
     inline size_t index_size() const { return _index_size; }
 
-    inline void set_to_max(char* buf) const { return _type_info->set_to_max(buf); }
+    virtual inline void set_to_max(char* buf) const { return _type_info->set_to_max(buf); }
+    //todo delete this method when segment v2 release
```

Comment:
```suggestion
    // TODO(wangbo) delete this method when segment v2 release
```
Should comment why this function stay here to let others know about this.

Reply:
👌